entries/301_show_notes.bacf

```diff
@@ -0,0 +1,72 @@
+- nr: 301
+- title: Live from CocoaHeads Hamburg
+- date: 14.01.2020, 20:00
+- file: 301_live_from_cocoaheads_hh.mp3
+- author: Bas Broek / Benedikt Terhechte
+- description: Where Bas still has no reason to buy new hardware, and Benedikt learned more about unicode than he wanted to. And both have an exciting year coming up. Live from CocoaHeads Hamburg on January 9, 2020.
+---
+<div>
+<p>
+  Please let us know your take-away of this episode by sharing it with us <a href="https://twitter.com/_contravariance">@_contravariance</a>.
+</p>
+<p>
+  An <a href="https://itunes.apple.com/us/podcast/contravariance-a-swift-podcast/id1423771323">iTunes review</a> is also greatly appreciated!
+</p>
+<p>
+  <h3>Contact information</h3>
+
+  <ul>
+    <li><a href="https://contravariance.rocks">Contravariance's website</a></li>
+    <li><a href="https://github.com/Contravariance/website">GitHub repository</a></li>
+    <li><a href="https://twitter.com/_contravariance">Contravariance's Twitter</a></li>
+    <li><a href="https://twitter.com/terhechte">Benedikt's Twitter</a></li>
+    <li><a href="https://twitter.com/basthomas">Bas' Twitter</a></li>
+  </ul>
+
+  <ul>
+    <li><a href="https://appventure.me">Benedikt's blog</a></li>
+    <li><a href="https://basthomas.github.io">Bas' blog</a></li>
+  </ul>
+</p>
+<p>
+  <h3>Hardware</h3>
+
+  <ul>
+    <li><a href="https://www.apple.com/mac-pro/">Mac Pro</a></li>
+    <li><a href="">Mac Pro wheels</a></li>
+    <li><a href="https://www.costco.com/Kirkland-Signature-Whole-Wheel-Parmigiano-Reggiano%2c-72-lbs.-.product.100096211.html">Parmigiano Reggiano wheel price</a></li>
+    <li><a href="https://www.apple.com/macbook-pro-16/">MacBook Pro 16 Inch</a></li>
+  </ul>
+</p>
+<p>
+  <h3>Secret Project</h3>
+
+  <ul>
+    <li><a href="https://droidkaigi.jp/2019/en/">DroidKaigi</a></li>
+    <li><a href="https://twitter.com/steveriggins/status/1215162339256700928">Swift 5.2 diagnostics</a></li>
+    <li><a href="https://twitter.com/terhechte/status/1211249990657548289">Benedits custom popover in SwiftUI</a></li>
```

Comment:
Can you double check this is the tweet you were references to in the episode, @terhechte?